src/components/EditModal/TagsInput.tsx

```diff
@@ -0,0 +1,76 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core"
+import * as React from "react"
+
+import { WordProperties } from "../../@core/api/model"
+
+import Input from "../../common/Input"
+import { StyledTag } from "../../common/Tag"
+
+type Props = {
+  word: WordProperties
+  disabled: boolean
+  onChange: (tags: string[]) => void
+}
+
+export const canAddTag = (currentTag: string, createdTags: string[]) => {
+  if (currentTag.length === 0) return false
```

Comment:
Is it the same as `!currentTag || !createdTags.includes(currentTag)`?

Reply:
Does `!currentTag` work because an empty string is falsy? I never dare to use the falsiness property because I can never remember what is truthy or falsy in JS. Shall I change it anyway?

Reply:
P.S.: Quickly tried to make a oneliner, and I would rather keep it the way it is with early exits. Maybe it's just me, but I find it tricky to wrap my head around a combination of `!` and `||` even in this simple case. Would be nice to have something like this though :)
```
match currentTag with
| "" -> false
| _ -> !createdTags.includes(currentTag)
```